frontend/packages/core/src/AppLayout/search.tsx

```diff
@@ -66,17 +119,24 @@ const Input = (params: AutocompleteRenderInputParams): React.ReactNode => {
     window.addEventListener("keydown", handleKeyPress);
   }, []);
 
-  const inputProps = { ...params.InputProps, style: { color: theme.palette.primary.main } };
-
   return (
     <InputField
       {...params}
-      label={<InputIcon />}
       placeholder="Search..."
-      variant="outlined"
       fullWidth
       inputRef={searchRef}
-      InputProps={inputProps}
+      InputProps={{
```

Comment:
if you're going to show more than 1 adornment, need to structure it like this

https://github.com/mui-org/material-ui/issues/19479